.github/workflows/lint.yml

```diff
@@ -0,0 +1,79 @@
+name: lint
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/**'
+  pull_request:
+
+env:
+  GO_VERSION: 1.23.x
+
+jobs:
+  go:
+    timeout-minutes: 5
+    name: "go | ${{ matrix.goos }} | ${{ matrix.gonext }}"
```

Comment:
"gonext" no longer exists

Reply:
Let me see if I can display something nicer than "true"

Reply:
What about the latest version?

`canary: go-canary`

Avoids the boolean literal issue, is consistent with the good notion of `canary` and does display "ok" in github UI checks?